### PR TITLE
Send Native Tokens across Chains Guide

### LINKED ISSUE #1608: Create a Guide on how to send Tokens across Chains
**Is your feature request related to a problem? Please describe.**

cwarnerdev has recently updated the repository we used as a base for the [Native Token and ERC20NativeToken how-to guide](https://wiki.iota.org/isc/how-tos/core-contracts/token/introduction/), Please review the recent changes to the [ISC cross-chain repo](https://github.com/iotaledger/isc-cross-chain) and update the following:

**Describe the solution you'd like**

Since it's possible to send the tokens across chains, please add an optional last step showing the users how to do this.

**Additional context**

Create a guide that shows developers how to transfer native tokens from the L1 native token to the L2 EVM account.

### PR DESCRIPTION
**Pull request by Ginowine**
# Description of change

This PR creates a guide on how to transfer native tokens from the L1 native token to the L2 EVM account. 

## Links to any relevant issues

fixes #1608.

## Type of change

- Documentation Enhancement

## Change checklist

Tick the boxes that are relevant to your changes, and delete any items that are not.

- [x] I have followed the [contribution guidelines](https://github.com/iota-wiki/iota-wiki/blob/main/.github/CONTRIBUTING.md) for this project
- [x] I have performed a self-review of my own changes
- [x] I have made sure that added/changed links still work
- [x] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

**Review by vivekjain23 — CHANGES_REQUESTED** [review]

File: docs/build/isc/v1.3-alpha/docs/how-tos/core-contracts/token/send-token-across-chains.mdx [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+        bytes memory destinationAddress,
+        uint256 amount,
+        ISCChainID chainID,
+        uint64 storageDeposit
+) external {
+    // Function implementation
+}
+```
+
+This function facilitates the transfer of a specified `amount` of tokens from the current chain to a `destinationChain`, with the `destinationAddress` as the recipient on the destination chain. The `sendCrossChain` function invokes the `ISC.sandbox.send` function, which manages the actual cross-chain message transmission.
```
Comment (salaheldinsoliman) [inline docs/build/isc/v1.3-alpha/docs/how-tos/core-contracts/token/send-token-across-chains.mdx:36]: Does it make sense to include the original function doc comments? 
Also, I think the term `wrapper function` is more descriptive, WDYT?

File: docs/build/isc/v1.3-alpha/docs/how-tos/core-contracts/token/send-token-across-chains.mdx [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+
+    function sendCrossChainMessage(
+        address destinationChain,
+        bytes memory destinationAddress,
+        uint256 amount,
+        ISCChainID chainID,
+        uint64 storageDeposit
+    ) external {
+        // Ensure the sender has enough tokens (assuming a balanceOf function exists)
+        uint256 senderBalance = IERC20(tokenAddress).balanceOf(msg.sender);
```
Comment (salaheldinsoliman) [inline docs/build/isc/v1.3-alpha/docs/how-tos/core-contracts/token/send-token-across-chains.mdx:101]: Lint: 
1- Import `IERC20` interface, as solc would complain when copy pasting the code
2- Either pass the token address as a parameter to `sendCrossChainMessage`, or indicate that it should be replaced/hardcoded

File: docs/build/isc/v1.3-alpha/docs/how-tos/core-contracts/token/send-token-across-chains.mdx [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+    }
+
+    function sendCrossChainMessage(
+        address destinationChain,
+        bytes memory destinationAddress,
+        uint256 amount,
+        ISCChainID chainID,
+        uint64 storageDeposit
+    ) external {
+        // Ensure the sender has enough tokens (assuming a balanceOf function exists)
```
Comment (salaheldinsoliman) [inline docs/build/isc/v1.3-alpha/docs/how-tos/core-contracts/token/send-token-across-chains.mdx:100]: ERC20 Standard always has a `balanceOf` function, so the assumption always works

**Review by salaheldinsoliman — CHANGES_REQUESTED** [review]
LGTM, just some minor comments

File: docs/build/isc/v1.1/docs/how-tos/core-contracts/token/send-token-across-chains.mdx [hunk 4]
```diff
@@ -0,0 +1,108 @@
...
+
+To send native tokens across chains, you need to provide the following transaction details to the `sendCrossChain` function. 
+
+* `ChainAddress` - The L1 address of the destination chain.
+* `Destination` - The address on the destination chain that will receive the tokens.
+* `chainID` - The ID of the destination chain.
+* `amount` - The amount of native tokens to sent.
+* `storageDeposit` - The base tokens to cover storage deposit.
+
+## Example Code
```
[the remark this answers is not in the document]
Comment (lucas-tortora) [reply docs/build/isc/v1.1/docs/how-tos/core-contracts/token/send-token-across-chains.mdx:52]: `##`  works here

File: docs/build/isc/v1.1/docs/how-tos/core-contracts/token/send-token-across-chains.mdx [hunk 5]
```diff
@@ -0,0 +1,108 @@
...
+<ExampleCodeIntro/>
+
+### 2. Send Token to Another Chain
```
[the remark this answers is not in the document]
Comment (lucas-tortora) [reply docs/build/isc/v1.1/docs/how-tos/core-contracts/token/send-token-across-chains.mdx:55]: `###` works here

File: docs/build/isc/v1.1/docs/how-tos/core-contracts/token/send-token-across-chains.mdx [hunk 6]
```diff
@@ -0,0 +1,108 @@
...
+  - native tokens
+  - cross-chain
+---
+import ExampleCodeIntro from '../../../_partials/how-tos/token/_example_code_intro.md';
+import DemoTokenSetup from '../../../_admonitions/_token-demo-setup.md';
+import CreateNativeToken from '../../../_admonitions/_create-native-token.md';
+
+# Send Native Token Across Chains
+
+In this guide, you'll learn how to send L1 native Tokens from L1 to an L2 EVM account on a destination chain by using the `sendCrossChain` function of the `NativeTokenController.sol` contract.
```
[the remark this answers is not in the document]
Comment (Ginowine) [reply docs/build/isc/v1.1/docs/how-tos/core-contracts/token/send-token-across-chains.mdx]: Thanks, done.

File: docs/build/isc/v1.3-alpha/docs/how-tos/core-contracts/token/send-token-across-chains.mdx [hunk 7]
```diff
@@ -0,0 +1,108 @@
...
+  - native tokens
+  - cross-chain
+---
+import ExampleCodeIntro from '../../../_partials/how-tos/token/_example_code_intro.md';
+import DemoTokenSetup from '../../../_admonitions/_token-demo-setup.md';
+import CreateNativeToken from '../../../_admonitions/_create-native-token.md';
+
+# Send Native Token Across Chains
+
+In this guide, you'll learn how to send L1 native Tokens from L1 to an L2 EVM account on a destination chain by using the `sendCrossChain` function of the `NativeTokenController.sol` contract.
```
[the remark this answers is not in the document]
Comment (Ginowine) [reply docs/build/isc/v1.3-alpha/docs/how-tos/core-contracts/token/send-token-across-chains.mdx]: Thanks, done.

File: docs/build/isc/v1.3-alpha/docs/how-tos/core-contracts/token/send-token-across-chains.mdx (hunk 1 above)
Comment (Ginowine) [reply docs/build/isc/v1.3-alpha/docs/how-tos/core-contracts/token/send-token-across-chains.mdx]: I modified the paragraph to implement your suggestions.

File: docs/build/isc/v1.3-alpha/docs/how-tos/core-contracts/token/send-token-across-chains.mdx (hunk 2 above)
Comment (Ginowine) [reply docs/build/isc/v1.3-alpha/docs/how-tos/core-contracts/token/send-token-across-chains.mdx]: Done! thanks

File: docs/build/isc/v1.1/docs/how-tos/core-contracts/token/send-token-across-chains.mdx (hunk 6 above)
[the remark this answers is not in the document]
Comment (Ginowine) [reply docs/build/isc/v1.1/docs/how-tos/core-contracts/token/send-token-across-chains.mdx]: Done, thanks.

File: docs/build/isc/v1.1/docs/how-tos/core-contracts/token/send-token-across-chains.mdx [hunk 8]
```diff
@@ -0,0 +1,108 @@
...
+        ISCChainID chainID,
+        uint64 storageDeposit
+    ) external {
+        // Ensure the sender has enough tokens (assuming a balanceOf function exists)
+        uint256 senderBalance = IERC20(tokenAddress).balanceOf(msg.sender);
+        require(senderBalance >= amount, "Insufficient token balance");
+
+        // Call the sendCrossChain function from NativeTokenController
+        nativeTokenController.sendCrossChain(destinationChain, destinationAddress, amount, chainID, storageDeposit);
+    }
```
[the remark this answers is not in the document]
Comment (Ginowine) [reply docs/build/isc/v1.1/docs/how-tos/core-contracts/token/send-token-across-chains.mdx]: Done, please confirm.

File: docs/build/isc/v1.1/docs/how-tos/core-contracts/token/send-token-across-chains.mdx [hunk 9]
```diff
@@ -0,0 +1,108 @@
...
+```
+
```
[the remark this answers is not in the document]
Comment (Ginowine) [reply docs/build/isc/v1.1/docs/how-tos/core-contracts/token/send-token-across-chains.mdx:77]: Thanks.

File: docs/build/isc/v1.1/docs/how-tos/core-contracts/token/send-token-across-chains.mdx [hunk 10]
```diff
@@ -0,0 +1,118 @@
...
+    require(senderBalance >= amount, "Insufficient token balance");
+
+    // Call the sendCrossChain function from NativeTokenController
+    nativeTokenController.sendCrossChain(destinationChain, destinationAddress, amount, chainID, storageDeposit);
+    }
+}
+```
+## Conclution
+
+By following this guide, you have learned how to set up your development environment and use the `sendCrossChain` function in the [`NativeTokenController.sol`](https://github.com/iotaledger/isc-cross-chain/blob/master/contracts/NativeTokenController.sol) contract to send native tokens across chains. You can now interact with the sendCrossChain function within your own smart contracts to facilitate cross-chain token transfers.
```
Comment (lucas-tortora) [inline docs/build/isc/v1.1/docs/how-tos/core-contracts/token/send-token-across-chains.mdx:116]:
```suggestion
By following this guide, you have learned how to set up your development environment and use the `sendCrossChain` function in the [`NativeTokenController.sol`](https://github.com/iotaledger/isc-cross-chain/blob/master/contracts/NativeTokenController.sol) contract to send native tokens across chains. You can now interact with the `sendCrossChain` function within your own smart contracts to facilitate cross-chain token transfers.
```

**Review by lucas-tortora — APPROVED** [review]
Other than some `` it LGTM :)

File: docs/build/isc/v1.1/docs/how-tos/core-contracts/token/send-token-across-chains.mdx (hunk 10 above)
Comment (Ginowine) [reply docs/build/isc/v1.1/docs/how-tos/core-contracts/token/send-token-across-chains.mdx]: Ok great! I have fixed it.

**Comment by Ginowine** [comment]
@lucas-tortora, I have made the suggested update.

**Review by lucas-tortora — APPROVED** [review]

**Review by salaheldinsoliman — APPROVED** [review]
@Ginowine  LGTM

File: docs/build/isc/v1.1/docs/how-tos/core-contracts/token/send-token-across-chains.mdx [hunk 11]
```diff
@@ -0,0 +1,118 @@
...
+    // Ensure the sender has enough tokens (assuming a balanceOf function exists)
+    uint256 senderBalance = IERC20(tokenAddress).balanceOf(msg.sender);
+    require(senderBalance >= amount, "Insufficient token balance");
+
+    // Call the sendCrossChain function from NativeTokenController
+    nativeTokenController.sendCrossChain(destinationChain, destinationAddress, amount, chainID, storageDeposit);
+    }
+}
+```
+## Conclution
```
Comment (lucas-tortora) [inline docs/build/isc/v1.1/docs/how-tos/core-contracts/token/send-token-across-chains.mdx:114]:
```suggestion
## Conclusion
```

**Pull request merged**